etc/disable-common.inc

```diff
@@ -117,8 +117,12 @@ read-only ${HOME}/.reportbugrc
 read-only ${HOME}/.xmonad
 read-only ${HOME}/.xscreensaver
 
-# The user ~/bin directory can override commands such as ls
+# Make directories commonly found in $PATH read-only
 read-only ${HOME}/bin
+read-only ${HOME}/.gem
+read-only ${HOME}/.local
```

Comment:
Yes, but those can make `${HOME}/.local/share/${foo}` read-writeable.
It's true, however, that this would require reviewing the current profiles  :(

The alternative would be enumerating a bunch of directories there, like `.local/{usr/,}{bin,lib,libexec}`, which has much weaker isolation but is highly unlikely to break current profiles.